src/nodeProvider.ts

```diff
@@ -22,10 +22,12 @@ export class DidactNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 	private _onDidChangeTreeData: vscode.EventEmitter<TreeNode | undefined> = new vscode.EventEmitter<TreeNode | undefined>();
 	readonly onDidChangeTreeData: vscode.Event<TreeNode | undefined> = this._onDidChangeTreeData.event;
 
-	protected treeNodes: TreeNode[] = [];
-	protected retrieveTutorials : boolean = true;
+	protected treeNodes: TreeNode[];
+	protected retrieveTutorials = true;
```

Comment:
is it normal that he type is removed?

Reply:
Yes, ESLint states that if you initialize a variable with a boolean value, then no type should be specified because it is implicit.

Reply:
Can you point to the rule please?

Reply:
![image](https://user-images.githubusercontent.com/435853/93449110-a4365200-f8d4-11ea-8e7e-94ab626f03fa.png)

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md